src/core/i18n/en/translation.en.json

```diff
@@ -1351,7 +1352,7 @@
       },
       "read-access": "Allow read access for non-space members to Subspaces",
       "about": "About this Space",
-      "visibility": "This Space is currently in <strong>{{visibility}}</strong> mode",
+      "visibility": "This Space is currently in <strong>{{mode}}</strong> mode",
```

Comment:
Do we need this chagne? It looks like it's used in SpaceProfileFixedInformation.tsx and it's not changed to mode there.

Reply:
I wish I knew earlier that there is already the same section.